spec/notifiers/publishing_api_notifier_spec.rb

```diff
@@ -0,0 +1,17 @@
+require "spec_helper"
+
+describe PublishingApiNotifier do
+  describe "#publish" do
+    before do
+      LicenceFinderContentItemPresenter.stub(:new).and_return(
```

Comment:
I think I like the fact that we're only testing `PublishingApiNotifier` here, instead of also implicitly testing the payload like we do in https://github.com/alphagov/business-support-finder/blob/master/spec/notifiers/publishing_api_notifier_spec.rb

But it feels a bit wrong to stub `:new` and to let this test know so much about the internals of the `PublishingApiNotifier`. Could we change `PublishingApiNotifier` to take the presenter as an argument?